config/initializers/active_merchant.rb

```diff
@@ -9,3 +9,21 @@
   ActiveMerchant::Billing::Gateway.wiredump_device = Rails.root.join('log/activemerchant.log').open('a')
   ActiveMerchant::Billing::Gateway.wiredump_device.sync = true
 end
+
+ActiveMerchant::Billing::StripeGateway.prepend(Module.new do
+  def headers(options = {})
+    key = options[:key] || @api_key
+    idempotency_key = options[:idempotency_key]
+
+    headers = {
+      'Authorization' => 'Basic ' + Base64.encode64(key.to_s + ':').strip.delete("\n"),
```

Comment:
```suggestion
      'Authorization' => 'Basic ' + Base64.encode64(key.to_s + ':').strip.chomp,
```

Reply:
It is not the same. I need it to remove it from the middle of the string XD